vllm/config.py

```diff
@@ -49,31 +51,32 @@ class ModelConfig:
             output). If None, will be derived from the model.
         quantization: Quantization method that was used to quantize the model
             weights. If None, we assume the model weights are not quantized.
-        enforce_eager: Whether to enforce eager execution. If True, we will
-            disable CUDA graph and always execute the model in eager mode.
-            If False, we will use CUDA graph and eager execution in hybrid.
-        max_context_len_to_capture: Maximum context len covered by CUDA graphs.
-            When a sequence has context length larger than this, we fall back
-            to eager mode.
+        flash_style: Enable flash style page attention. This is only supported
+            by llama models.
+        max_chunked_prefill_len: The maximum length of tokens for prefill
+            requests. Longer requests will be chunked into multiple chunks.
+            -1 means no chunking (disabled). This features is only supported
+            for flash style attention.
     """
 
-    def __init__(
-        self,
-        model: str,
-        tokenizer: str,
-        tokenizer_mode: str,
-        trust_remote_code: bool,
-        download_dir: Optional[str],
-        load_format: str,
-        dtype: Union[str, torch.dtype],
-        seed: int,
-        revision: Optional[str] = None,
-        tokenizer_revision: Optional[str] = None,
-        max_model_len: Optional[int] = None,
-        quantization: Optional[str] = None,
-        enforce_eager: bool = False,
-        max_context_len_to_capture: Optional[int] = None,
-    ) -> None:
+    def __init__(self,
+                 model: str,
+                 tokenizer: str,
+                 tokenizer_mode: str,
+                 trust_remote_code: bool,
+                 download_dir: Optional[str],
+                 load_format: str,
+                 dtype: Union[str, torch.dtype],
+                 seed: int,
+                 revision: Optional[str],
+                 tokenizer_revision: Optional[str] = None,
+                 max_model_len: Optional[int] = None,
+                 quantization: Optional[str] = None,
+                 enable_cuda_graph: bool = False,
+                 cuda_graph_max_context_len: int = 5000,
+                 cuda_graph_cache_size: int = 10,
+                 flash_style: bool = False,
+                 max_chunked_prefill_len: int = -1,
```

Comment:
) ?

Reply:
```
vllm-public/vllm/engine/async_llm_engine.py", line 7, in <module>
    from vllm.anyscale.lora.utils import LoRARequest
ModuleNotFoundError: No module named 'vllm.anyscale'
```

Reply:
Having the same issue. Any update on this?